src/main/java/com/amida/saraswati/jdbcsink/model/AddressIngest.java

```diff
@@ -0,0 +1,76 @@
+package com.amida.saraswati.jdbcsink.model;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+//@Entity
+public class AddressIngest {
+
+	@JsonProperty("firstName")
```

Comment:
I think the json property should be street?

Reply:
Also since you are using the same name as the variable is there a reason for JsonProperty?